### ISSUE TITLE
Unable to locate element: {"method":"css selector","selector":".BtnTxt"}

### ISSUE THREAD

**Issue by caibinglin** [Konyanj0278/BestBuy-Automated-Checkout#14]
Unable to locate element: {"method":"css selector","selector":".BtnTxt"}
Is it caused by the change of bestbuy official website？

**Comment by caibinglin** [comment]
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":".c-modal-close-icon"}
  (Session info: chrome=91.0.4472.77)

**Comment by Konyanj0278** [comment]
Yes it is, I was banned from making online purchases thanks to my vpn. So I haven’t been able to make any changes and test them out.

**Comment by caibinglin** [comment]
Manual login of chrome browser to bestbuy website is also a prompt to report an error.I can only use the mobile app to log in.

**Comment by caibinglin** [comment]
I can give you a new free VPN,if you need.

**Comment by caibinglin** [comment]
it is too hard to buy video cards

**Comment by Konyanj0278** [comment]
Ohh for video cards I don’t think my method would work, from last I knew there is a human verification method.  I’ll try to update it for regular products but the way BestBuy is handling graphics cards I don’t see a way to automate it without completely changing the way the bot works.

**Comment by caibinglin** [comment]
Besides the graphics card, I also want to snap up some discounted products.

**Comment by Konyanj0278** [comment]
I’ll try updating it some time soon
